packages/nextcloud/lib/src/api/core.openapi.json

```diff
@@ -3560,12 +3560,6 @@
                                                                 {
                                                                     "$ref": "#/components/schemas/CommentsCapabilities"
                                                                 },
-                                                                {
-                                                                    "$ref": "#/components/schemas/CoreCapabilities"
-                                                                },
-                                                                {
-                                                                    "$ref": "#/components/schemas/CorePublicCapabilities"
-                                                                },
```

Comment:
how is this change not reflected in the generated dart code 🤔

Reply:
This was the problem with the duplicate entries you found a while back. We just didn't update the specs/openapi-extractor back then exactly because it doesn't change the generated code (the duplicate entries are right above the highlighted diff).

Reply:
Ah I remeber that they where just duplicates ...